Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -193,223 +193,223 @@
     <!-- ####################################
                Risk Details
     ###################################### -->
-    <string name="risk_details_headline_behavior">Verhalten Sie sich richtig</string>
-    <string name="risk_details_behavior_body_stay_home">Bleiben Sie zu Hause</string>
-    <string name="risk_details_behavior_body_avoid_contact">Vermeiden Sie sozialen Kontakt</string>
-    <string name="risk_details_behavior_body_contact_doctor">Für Fragen zu auftretenden Symptomen, Testmöglichkeiten und weiteren Absonderungsmaßnahmen wenden Sie sich bitte an:</string>
-    <string name="risk_details_behavior_body_wash_hands">Waschen Sie Ihre Hände</string>
-    <string name="risk_details_behavior_body_wear_mask">Tragen Sie eine Schutzmaske</string>
-    <string name="risk_details_behavior_body_stay_away">Halten Sie 2 Meter Abstand zu anderen Personen</string>
-    <string name="risk_details_behavior_body_cough_sneeze">Husten und Niesen Sie in Ihre Armbeuge</string>
-    <string name="risk_details_behavior_increased_body_1">- Ihren Hausartzt/Ihre Hausärtztin</string>
-    <string name="risk_details_behavior_increased_body_2">- Den Kassenärtzlichen Notdienst (Tel: 116117)</string>
-    <string name="risk_details_behavior_increased_body_3">- Ihr Gesundheitsamt</string>
-    <string name="risk_details_headline_infection_risk">Infektionsrisiko</string>
-    <string name="risk_details_body_infection_risk">So wird Ihr Risiko ermittelt</string>
+    <string name="risk_details_headline_behavior">Behave correctly</string>
```

Comment:
I'm not sure what it means. But "Take appropriate actions" seems more appropriate here.

Reply:
Same here. I would leave it like this